### DISCUSSION TITLE
Is it possible to make a themes plugin?

### DISCUSSION THREAD

**Discussion by solarkite** [koreader/koreader#8527] [Ideas]
Is it possible to make a themes plugin similar to the kindle's new themes selection. Generally, for night and day operation, i.e. night mode. Also book specific, such as selecting a font to be used by a specific book.

**Comment by Frenzie** [comment]
See tools → profiles.

**Reply by solarkite** [reply]
> See tools → profiles.

Thanks for responding, Frenzie. I’ve looked at profiles. Maybe the feature sets are too overwhelming, but I don’t see how I would implement the examples I gave. I’ve set up a night mode profile and alternatively a day made profile. However I don’t see a selection to change the type of font used. Nor a way to make it book specific. I also wish it was more quickly accessible rather then buried in the settings.

**Reply by Frenzie** [reply]
All settings are per book unless you make them default by long pressing.

If you can't set a font through the profile thingy, that might be a missing feature.

**Reply by poire-z** [reply]
> Nor a way to make it book specific.

Most settings stick per book, so there's no reason to have anything to make settings specific to book. May be nightmode is a global one (because well, it depends on time of day rather than a book ;).
There is also a docsettigstweak plugin, that may apply different defaults to new book opening depending on which directories they are in.

> I also wish it was more quickly accessible rather then buried in the settings.

I think any profiles you create is available in the Gesture menu, so you can assign a profile to a gesture (tap in corner, swipe on the edge, multiswipes...)
